lib/manager/composer/extract.js

```diff
@@ -0,0 +1,45 @@
+
+module.exports = {
+  extractDependencies,
+};
+
+function extractDependencies(content, packageFile) {
+  logger.debug('composer.extractDependencies()');
+  let packageJson;
+  try {
+    packageJson = JSON.parse(content);
+  } catch (err) {
+    logger.info({ packageFile }, 'Invalid JSON');
+    return null;
+  }
+  const deps = [];
+  const depTypes = [
+    'require',
+    'require-dev',
+  ];
+  for (const depType of depTypes) {
+    if (packageJson[depType]) {
+      try {
+        for (const [depName, version] of Object.entries(packageJson[depType])) {
+          if (depName.includes('/')) {
+            deps.push({
+              depName,
+              depType,
+              currentVersion: version.trim().replace(/^/, ''),
+            });
```

Comment:
We should also recorded the `lockedVersion` here for each dep too, if lockfile exists